Cargo.toml

```diff
@@ -8,4 +8,8 @@ members = [
     "owner-onboarding-service",
     "owner-tool",
     "rendezvous-server",
+    "manufacturing-server",
+    "manufacturing-client",
+
```

Comment:
just curious, why the empty line?

Reply:
Because the integration-tests crate isn't intended to be shipped.